src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -875,7 +875,7 @@ def makeSubmit(self, jobList):
             jdl.append("+JobLeaseDuration = isUndefined(MachineAttrMaxHibernateTime0) ? 1200 : MachineAttrMaxHibernateTime0\n")
 
             ### print all the variables needed for us to rely on condor userlog
-            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,JobStartDate,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
+            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
```

Comment:
`JOBGLIDEIN_CMSSite` seems to be an expression using `GLIDEIN_CMSSite` which turns to be undefined. E.g. from vocms0303:
```
cmst1@vocms0303:/data/srv/wmagent/current $ condor_q -af:h CMS_JobType JobStatus MATCH_EXP_JOBGLIDEIN_CMSSite JOBGLIDEIN_CMSSite GLIDEIN_CMSSite -limit 2
CMS_JobType JobStatus MATCH_EXP_JOBGLIDEIN_CMSSite JOBGLIDEIN_CMSSite                                                         GLIDEIN_CMSSite
Processing  2         T2_FR_IPHC                   $$([ifThenElse(GLIDEIN_CMSSite is undefined, "Unknown", GLIDEIN_CMSSite)]) undefined      
Production  2         T1_RU_JINR                   $$([ifThenElse(GLIDEIN_CMSSite is undefined, "Unknown", GLIDEIN_CMSSite)]) undefined      
```

About `EnteredCurrentStatus`, I'll keep it around just to be explicit that it's supposed to be in the condor log.